Amz.py

```diff
@@ -222,8 +236,13 @@ def main():
             elif 'you won!' in title.text:
                 print(title.text)
                 print('You won!')
-                you_won = True
-                break
+                # Call the confirm_address function
+                if confirm_address(driver):
+                    # If you win, will continue; exit if issue occurred.
+                    time.sleep(15)
+                    continue
+                else:
+                    break
```

Comment:
replace `break` with `return True` here

Reply:
fixed in next commit: https://github.com/Penlo/Amazon-Giveaway-Bot/pull/12/commits/856a575e6d8d20a617488f7e5daf721838e571f4